packages/usa-in-page-navigation/src/index.js

```diff
@@ -162,6 +165,10 @@ const createInPageNav = (inPageNavEl) => {
 
     inPageNavList.appendChild(listItem);
     listItem.appendChild(navLinks);
+
+    if (window.location.hash === linkHref) {
+      handleScrollToSection(anchorTag);
+    }
```

Comment:
Thoughts on moving this into it's own function and call it after the component has been created?

Something similar to `handleClickFromLink()`. It might help improve clarity.

Reply:
@mejiaj Are you thinking it would be something we would call in the behavior's `init`, like `scrollToCurrentSection`, where the function would try to find and scroll to the heading associated with the current URL fragment?

Reply:
@aduth yes, that seemed like the best place at the time of original review. Maybe I misunderstood the intent.

Reply:
I think that could make sense to separate. I pushed up some changes in 786d92e8d. Is that what had in mind?